src/layers/core/hexagon-layer/hexagon-layer-vertex.glsl

```diff
@@ -51,12 +52,13 @@ void main(void) {
 
   // calculate elevation, if 3d not enabled set to 0
   // cylindar gemoetry height are between -0.5 to 0.5, transform it to between 0, 1
-  float elevation =  mix(0., project_scale(instancePositions.z  * (positions.y + 0.5) * ELEVATION_SCALE), enable3d);
+  float elevation =  mix(0., project_scale(instancePositions.z  * (positions.y + 0.5) * ELEVATION_SCALE), extruded);
```

Comment:
As an alternative, we could also generate a simpler primitive geometry when extruded=false, i.e. not have any side triangles.

This could impact the hexagon layer rendering performance as there would be less invocations of the vertex shader in non-extruded mode.

I don't have a strong opinion here, other than that we might want this to work somewhat similarly between layers.